dotCMS/src/main/webapp/html/portlet/ext/contentlet/publishing/view_publish_audit_detail.jsp

```diff
@@ -24,6 +27,15 @@
 
     Bundle bundle = APILocator.getBundleAPI().getBundleById(bundleId);
 
+    List<String> endpointsProtocols = new ArrayList<>();
+    try {
+        for (Environment e: APILocator.getEnvironmentAPI().findEnvironmentsByBundleId(bundleId)) {
+            pepAPI.findSendingEndPointsByEnvironment(e.getId()).forEach(endpoint -> {
```

Comment:
Let's rename pepAPI to something easy to understand.

Reply:
Done https://github.com/dotCMS/core/pull/27364/commits/0c4e5e64e4b690bd6461847351c5c7aace6b86cc